vcluster/deploy/environment/aks.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Deploy on AKS
+sidebar_label: AKS
+sidebar_position: 8
+id: aks
+description: Learn how to deploy vCluster on Azure Kubernetes Service (AKS), including storage provisioning and managed identity configuration.
+---
+
+import InstallCli from '../../_partials/deploy/install-cli.mdx';
+
+<!-- vale off -->
+# Deploy vCluster on AKS
+<!-- vale on -->
+
+This guide provides step-by-step instructions for deploying vCluster on [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/products/kubernetes-service).
+
+## Prerequisites
+
+Before starting, ensure you have the following tools installed:
+
+- `kubectl` installed: Kubernetes command-line tool for interacting with the cluster. See [Install and Set Up kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) for installation instructions.
+- `vCluster CLI` <InstallCli />
+- [Azure CLI (az)](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+  :::note
+  Ensure you have the necessary permissions to create and manage AKS clusters in your Azure subscription.
+  :::
+
+## Create AKS cluster
+
+:::tip Right subscription
+Ensure you are using the correct Azure subscription by running `az account show`.
+:::
+
+Start by creating an AKS cluster using the Azure CLI. First, set up your environment variables:
+
+```bash title="Set environment variables"
+export RESOURCE_GROUP=vcluster-demo-rg
+export CLUSTER_NAME=vcluster-demo
+export LOCATION=eastus
+export NODE_SIZE=Standard_D4s_v3
+export NODE_COUNT=2
+```
+
+Create a resource group:
+
+```bash title="Create resource group"
+az group create --name $RESOURCE_GROUP --location $LOCATION
+```
+
+Create the AKS cluster:
+
+```bash title="Create AKS cluster"
+az aks create \
+    --resource-group $RESOURCE_GROUP \
+    --name $CLUSTER_NAME \
+    --location $LOCATION \
+    --node-count $NODE_COUNT \
+    --node-vm-size $NODE_SIZE \
+    --enable-managed-identity \
+    --enable-addons monitoring \
+    --generate-ssh-keys
+```
+
+This command creates an AKS cluster with managed identity enabled and Azure Monitor for containers.
+
+### Configure kubectl
+
```

Comment:
I just merged the EKS docs, but how come we don't give docs on how to configure kubectl for EKS?

Reply:
There is no special configuration needed and `kubectl` is [mentioned in the EKS prerequisites](https://www.vcluster.com/docs/vcluster/next/deploy/environment/eks#prerequisites).